### DISCUSSION TITLE
Filter based on Other Tags present in VCF

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#3962] [General]
Hi Team,

While trying to filter based on the Call Quality - "High Quality" is only GQ and AB considered as variants with low read depth also comes up? Is there any possibility to filter based on "Read Depth" or any other tags like VF, NL, SB, NC, US, AQ, LQ? If not can a feature be added to filter the variants with the tags present in the VCF file?

Thanks and Regards,
Tulasi S

**Comment by hanars** [comment]
Hi Tulsai,

Our team does not believe that filtering on Read Depth provides any substantial added utility for searches, and that setting meaningful GQ filters is the most effective way to filter out low quality data. As such, we have no plans to add filtration on this in seqr. The other tags you mention are not even available in most VCFs loaded to seqr and are not stored in the seqr data, so it is quite impossible to filter on those fields as well. 

If you really want to be able to filter based on arbitrary quality fields, you can add your own pre-processing logic to populate the "filters" field of the VCF. The "pass" quality filter will not return any variants that have any "filters", so if you want to add "filters" based on failed cutoffs to the VCF before loading that would result in the "pass" filter excluding those variants.

**Comment by DanizKooshavar** [comment]
I would also use a "read depth" filter on seqr. It would come in handy, especially in WES analysis.
Currently, a permissive search for de novo AD variants on a trio WES results in ~400 variants, of which over 80% have a read depth of below 15 (mostly below 10). It would be beneficial to filter out the low read-depth variants on a WES in the initial stages of variant curation.

**Reply by hanars** [reply]
Read depth is not helpful in standardized searching because it is so specific to the exact sequencing product that was used, which is why we allow filtering on normalized quality scores like GQ
